src/enrichment.cc

```diff
@@ -255,16 +256,19 @@ void Enrichment::GetMatlTrades(
                           cyclus::Material::Ptr> >& responses) {
   using cyclus::Material;
   using cyclus::Trade;
+  using cyclus::toolkit::RecordTimeSeries;
 
   intra_timestep_swu_ = 0;
   intra_timestep_feed_ = 0;
 
-  std::vector<Trade<Material> >::const_iterator it;
+  double tails_supply_qty = 0;
+  double prod_supply_qty = 0;
```

Comment:
Ditto: why create this extra tracker data member when one could just as easily query the current quantity of the prod object at any time?